src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Resources.resx

```diff
@@ -129,4 +129,7 @@
   <data name="ForegroundDispatcher_NoMethodNamePlaceholder" xml:space="preserve">
     <value>The method</value>
   </data>
+  <data name="JoinableTaskContextExtensions_AssertUIThread" xml:space="preserve">
```

Comment:
fwiw, I don't believe Roslyn localizes error messages (or at least, I haven't been doing that, lol).

Reply:
Actually I just checked and I was totally wrong, Roslyn _does_ localize certain error messages, at least the yellow bar ones (e.g. "so-and-so code fix has been disabled"). However, specific details about what went wrong (i.e. stack trace and specific error message) don't seem to get localized.

Reply:
In our tooling stack we don't currently present any yellow bars directly so I'd actually say we should avoid localizing these if possible because we'll be the ones needing to decipher the error messages in reported issues 😄

Reply:
Sounds good! I removed the added ones in my latest push, but the left the existing ones for now.